src/components/Dropdown/Dropdown.st.css

```diff
@@ -153,6 +280,8 @@
 
 .divider {
     -st-extends: Divider;
+    opacity: 30%;
+    background-color: value(CurrentItemTextColor);
```

Comment:
I think you'll need to use the st-mixin approach here as well 😬 
https://github.com/wix/wix-ui-tpa/blob/master/src/components/Divider/Divider.st.css#L18

Reply:
Also I think the -st-extends isn't necessary here